src/main/scala/nl/knaw/dans/easy/bagindex/command/CommandLineOptions.scala

```diff
@@ -59,26 +59,20 @@ class CommandLineOptions(args: Array[String], properties: PropertiesConfiguratio
   footer("")
 
   object interaction {
-    /**
-     * Interactive command line interface to ask whether the user is sure to index the whole bag-store
-     * and thereby delete all current data in the bag-index.
-     *
-     * @return `true` if the user wants to proceed, `false` otherwise
-     */
     def deleteBeforeIndexing(): Boolean = {
       @tailrec
       def recursiveAsk(): Boolean = {
+        print("Proceed? (y/n): ")
         StdIn.readLine().toLowerCase match {
-          case "yes" => true
-          case "no" => false
+          case "y" => true
+          case "n" => false
           case _ =>
-            println("either use 'Yes' or 'No'")
+            println("Answer y or n.")
             recursiveAsk()
         }
       }
 
-      println("Before the bag-store is indexed, the current index needs to be deleted?")
-      println("Do you want to proceed? (Yes/No)")
+      println("WARNING: you are about to rebuild the entire index. You should NOT attempt this while the daemon is running.")
```

Comment:
In general I think the solution is a combination of both retry policies (for example with random or exponential back-off) and transaction control. In this context, you might find [Hystrix](https://github.com/netflix/hystrix) an interesting solution...

For now letting the deposit fail seems in line with our current 'theme', but on the other hand, we do warn the user that it might take a while (maybe a week?) to have the deposit processed. Therefore, putting it back on the queue for later depositing might be a good option as well. But that requires some more architectural thinking and is not fitting to discuss here. For now I'll accept this statement and will proceed to merge this PR into mine.